docs/neo-n3/dapp_development/wallets_and_accounts.md

```diff
@@ -16,115 +14,68 @@ The easiest way to create a new wallet is by using one of the static creation me
-the `label(String)` method.
+An account can hold a label that is set to its address by default. If needed, you can change it to a custom label using the `label(String)` method.
 
 ```java
 Account a = Account.fromWIF("L3kCZj6QbFPwbsVhxnB8nUERDy4mhCSrWJew4u5Qh5QmGMfnCTda")
         .label("MyAccount");
 ```
 
-> **Note:** When you want to extract a single account instance, e.g. as a data transfer object (dto) make sure to create a
-> `NEP6Account` instance and use this instance. To do so, you will have to encrypt the account first and then call
-> `toNEP6Account`. This conversion is automatically made for each account when creating a `NEP6Wallet` from a wallet instance.
+> **Note:** When you want to extract a single account instance, for example as a data transfer object (DTO), make sure to create a `NEP6Account` instance and use this instance. To do so, you will need to encrypt the account first and then call `toNEP6Account`. This conversion is automatically performed for each account when creating a `NEP6Wallet` from a wallet instance.
 
 ## Multi-sig Accounts
 
 Multi-signature accounts can be created with the following methods:
 
 - `fromVerificationScript(VerificationScript script)`
-- `fromNEP6Account(NEP6Account nep6Acct)`
+- `fromNEP6Account(NEP6Account nep6Account)`
 - `createMultiSigAccount(List<ECPublicKey> publicKeys, int signingThreshold)`
-- `createMultiSigAccount(String address, int signingThreshold, int nrOfParticipants)`
-
-The first two methods can be used for single- and multi-sig accounts. The verification script - which is available in
-the NEP6Account's script - holds all information needed about the multi-sig account. This includes the signing
-threshold, the number of participants, and the involved public keys.  
-In the latter two methods there is now verification script available. Therefore, one or both of signing threshold and
-number of participants has to be specified explicitly in order such that the multi-sig account can be used as a signer in
-transactions. The `TransactionBuilder` needs the signing threshold and number of participants to determine the network
-fee that has to be paid for signing with the account.
-
-Multi-sig accounts do not hold EC key pairs. That would defeat the purpose of multi-sig accounts, because
-their key material should be spread over multiple entities.
-
-In the following example a new mutli-sig account is created from three public keys. Its signing threshold is two, i.e.,
-for a transaction signed by this account to be successful, at least two of the three participants have to sign.
-
-```java
-List<ECPublicKey> publicKeys = Arrays.asList(
-        ECKeyPair.createEcKeyPair().getPublicKey(),
-        ECKeyPair.createEcKeyPair().getPublicKey(),
-        ECKeyPair.createEcKeyPair().getPublicKey()
-);
-
-Account a2 = Account.createMultiSigAccount(publicKeys, 2)
-        .label("MyMultiSigAccount");
-```
-
```

Comment:
maybe I got distracted at the end of the review, BUT:

Did you delete all this content? don't you think this is useful? ☝️